plugins/destination/postgresql/cloud-config-ui/src/form/radio.tsx

```diff
@@ -0,0 +1,90 @@
+import { ReactNode } from 'react';
+
+import Radio from '@mui/material/Radio';
+import Stack from '@mui/material/Stack';
+import useTheme from '@mui/material/styles/useTheme';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import Typography from '@mui/material/Typography';
+
+interface Props<Value extends string | number | boolean> {
+  /** Callback that is called when the selected value changes. */
+  onChange: (newValue: Value) => void;
+  /** The currently selected value. */
+  value: Value;
+  /** The title of the radio group. */
+  title?: string;
+  /** The radio buttons to display. */
+  items: Array<{
+    title: ReactNode;
+    subtitle?: ReactNode;
+    value: string | number | boolean;
+    disabled?: boolean;
+  }>;
+}
+
+/**
+ * This component displays a group of radio buttons.
+ *
+ * @public
+ */
+export function RadioGroupSelector<Value extends string | number | boolean>({
```

Comment:
we have ExclusiveToggle in the plugin-config-ui-lib that is pretty close to this, not sure if you want to swap it out?

Reply:
Thanks, this is actually leftover